.npmrc

```diff
@@ -1,2 +1,3 @@
 save-exact = true
 engine-strict = true
+legacy-peer-deps = true
```

Comment:
This is supposed to be a temporary solution until we fix all the peer dependency issues.

Reply:
@kevin940726 I created [a draft PR](https://github.com/WordPress/gutenberg/pull/48990) that solves the peer dependency issue related to React Native and removes this NPM configuration option. As far as I tested, the dependency installation works, although warnings are displayed. Let me know if you'd like to incorporate it into this PR, thanks 🙇 !

Reply:
Nice! I tried the same thing too, but I think it would be better to be a separate follow-up PR. Warnings can be intimidating for some, I would like to solve all warnings before we roll them out to all contributors.